examples/maven-spring-custom/app/pom.xml

```
@@ -32,12 +32,12 @@
             <plugin>
                 <groupId>community.flock.wirespec.plugin.maven</groupId>
                 <artifactId>wirespec-maven-plugin</artifactId>
-                <version>0.0.0-SNAPSHOT</version>
+                <version>${wirespec.version}</version>
                 <dependencies>
                     <dependency>
                         <groupId>community.flock.wirespec.example.maven</groupId>
                         <artifactId>emitter</artifactId>
-                        <version>0.0.0-SNAPSHOT</version>
+                        <version>${wirespec.version}</version>
```

This refers to a module within your example, not the version of wirespec
```suggestion
                        <version>0.0.0-SNAPSHOT</version>
```